starter-code/index.html

```diff
@@ -1,42 +1,97 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Spotify</title>
-</head>
-<body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Spotify</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="nav-cont">
+        <img
+          class="nav-img"
+          src="./images/spotify-logo.png"
+          alt="spotify logo"
+        />
+        <ul class="navbar-content">
+          <li>Premium</li>
```

Comment:
These should be in a tags because they would link you to other parts of the site